### ISSUE TITLE
Alternate Default Project Image Loading

### ISSUE THREAD

**Issue by mmr6424** [LookingforGrp-rit/LookingForGroup#267]
Currently, projects (and profiles I believe), do not have a default alternative if the image FAILS to load (AKA a file was moved, deleted, changed, etc.). Instead, it shows "Project Image" alt text. This should be fixed.

**Comment by bcorie** [comment]
Waiting for PR review
